### ISSUE TITLE
Support for aarch64/RockPi 4

### ISSUE THREAD

**Issue by BTopbas** [i96751414/torrest#1]
I upgraded my Raspberry Pi to RockPi (RK3399 Processor) device. I tried the linux-arm64 binary and looks like working(because creates downloads and torrents folders successfully) but crashes immediately.  

When I run torrest. Here is the output.
```
INFO  main         - main             Loading configs
INFO  main         - main             Starting bittorrent service
INFO  bittorrent   - configure        Applying session settings...
INFO  bittorrent   - configure        UserAgent: torrest/0.0.4 libtorrent/1.2.9.0
INFO  main         - main             Starting torrent daemon on port 8080
Segmentation fault
```

**Comment by i96751414** [comment]
Any chance you can provide a backtrace?

**Comment by BTopbas** [comment]
How?

**Comment by i96751414** [comment]
Before that, can you try this binary? [torrest.zip](https://github.com/i96751414/torrest/files/5336833/torrest.zip)

**Comment by BTopbas** [comment]
Works! Thank you!!

**Comment by i96751414** [comment]
Nice. I will make it official soon.

**Comment by i96751414** [comment]
Fixed in [v0.0.5](https://github.com/i96751414/torrest/releases/tag/v0.0.5).
